client/my-sites/plugins/plugins-browser-item/index.jsx

```diff
@@ -189,21 +193,39 @@ const InstalledInOrPricing = ( {
 	currentSites,
 } ) => {
 	const translate = useTranslate();
+	const currentSite = currentSites.length === 1 ? currentSites[ 0 ] : null;
+	const installedPlugins =
+		useSelector( ( state ) => getInstalledPlugins( state, [ currentSite?.ID ] ) ) || [];
+	const activePlugins = installedPlugins.filter(
+		( activePlugin ) => activePlugin.sites[ currentSite?.ID ]?.active
+	);
```

Comment:
Can we avoid running this code when `selectedSiteId` is `null` or `undefined`?

Reply:
We should be able to avoid the calculation on `activePlugins` when there is no selected site.
But the `installedPlugins` should occur every time as per the rule of hooks: [Only Call Hooks at the Top Level](https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level).

Reply:
exactly!